.env.prod.yaml

```diff
@@ -9,7 +9,6 @@ GOOGLE_UPLOAD_BUCKET: "cidc-uploads-prod"
 GOOGLE_DATA_BUCKET: "cidc-data-prod"
 GOOGLE_LOGS_BUCKET: "cidc-logs-export-prod"
 GOOGLE_EMAILS_TOPIC: "emails"
-GOOGLE_CLOUD_PROJECT: "cidc-dfci"
```

Comment:
This value is provided by Cloud Functions in the `GCP_PROJECT` env variable automatically.